cluster/images/etcd/rollback/rollback.go

```diff
@@ -26,6 +26,8 @@ import (
 	"strings"
```

Comment:
@mikedanese 

OK. Now I see them... The default output from build-log.txt is very confusing since it hides all valuable output with `... skipping 1747 lines ...` and printed out tons of unrelated errors with yellow highlights.

```
W1215 09:44:40.288] I1215 09:44:40.287939   10387 parse.go:353] type checker error: /usr/local/go/src/runtime/trace.go:139:30: cannot use (traceBufHeader literal) (value of type traceBufHeader) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
W1215 09:44:40.289] I1215 09:44:40.288070   10387 parse.go:353] type checker error: /usr/local/go/src/runtime/trace.go:139:7: array length 64 << 10 - unsafe.Sizeof((traceBufHeader literal)) (value of type uintptr) must be constant
W1215 09:44:40.289] I1215 09:44:40.288156   10387 parse.go:353] type checker error: /usr/local/go/src/runtime/mgcwork.go:285:37: cannot use (workbufhdr literal) (value of type workbufhdr) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
W1215 09:44:40.289] I1215 09:44:40.288181   10387 parse.go:353] type checker error: /usr/local/go/src/runtime/mgcwork.go:285:7: array length (_WorkbufSize - unsafe.Sizeof((workbufhdr literal))) / sys.PtrSize (value of type uintptr) must be constant
W1215 09:44:40.289] I1215 09:44:40.288556   10387 parse.go:353] type checker error: /usr/local/go/src/runtime/chan.go:21:28: cannot use (hchan literal) (value of type hchan) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
W1215 09:44:40.289] I1215 09:44:40.288590   10387 parse.go:353] type che
```

We probably need to improve this output format to make it less confusing.

I will play with bazel more tomorrow.